src/routes/+layout.svelte

```diff
@@ -2,7 +2,7 @@
 <script lang="ts">
 	// Dependency: Highlight JS
 	import hljs from 'highlight.js';
-	import '$lib/styles/highlight-js.css'; // was: 'highlight.js/styles/github-dark.css';
+	import '$docs/styles/highlight-js.css'; // was: 'highlight.js/styles/github-dark.css';
```

Comment:
The comment should now be `// was: '$lib/styles/highlight-js.css'`, or be removed - is it important what it used to be?

Reply:
The comment isn't really useful any more so I'm dropping it completely.